src/EventStore.Core/Index/IndexStatusTracker.cs

```diff
@@ -14,12 +16,15 @@ public class IndexStatusTracker : IIndexStatusTracker {
 			_metric = new("Index", metric);
 		}
 
+		public IDisposable StartInitializing() => _metric.StartActivity("Initializing");
```

Comment:
perhaps call this one 'Opening' to distinguish it from the stream existence filter initialization